ibis/pandas/aggcontext.py

```diff
@@ -287,48 +292,71 @@ def compute_window_spec_default(obj, _):
 
 
 class Window(AggregationContext):
-
     __slots__ = 'construct_window',
 
     def __init__(self, kind, *args, **kwargs):
```

Comment:
can you not list these args directly?

Reply:
No, because these are potentially going into a function called in `apply`.